### ISSUE TITLE
dotnet run in directory with space doesn't work

### ISSUE THREAD

**Issue by pakrym** [dotnet/cli#319]
It treats part after space as a parameter.

```
C:\Users\user\Documents\visual studio 2015\Projects\TryMvc\src\TryMvc> dotnet run
Unrecognized command or argument 'studio'
```

**Comment by muratg** [comment]
Same on OS X.

```
Murats-MacBook-Pro% pwd
/Users/mg/Src/dotnet/my app
Murats-MacBook-Pro% dotnet run
Unrecognized command or argument 'app'
```

**Comment by davidfowl** [comment]
Dupe of #93

**Comment by muratg** [comment]
This repro's with a a few days old build.

**Comment by brthor** [comment]
Will try to repro this after fix for #525 comes in, expecting it is the same issue.

**Comment by brthor** [comment]
This should be resolved via #985, please reopen if you are still hitting this issue.
